librz/core/cmd_analysis.c

```diff
@@ -8724,159 +8706,11 @@ static int cmd_analysis_all(RzCore *core, const char *input) {
 				goto jacuzzi;
 			}
 			rz_cons_clear_line(1);
-			bool cfg_debug = rz_config_get_i(core->config, "cfg.debug");
 			if (*input == 'a') { // "aaa"
-				if (rz_str_startswith(rz_config_get(core->config, "bin.lang"), "go")) {
-					oldstr = rz_print_rowlog(core->print, "Find function and symbol names from golang binaries (aang)");
-					rz_print_rowlog_done(core->print, oldstr);
-					rz_core_analysis_autoname_all_golang_fcns(core);
-					oldstr = rz_print_rowlog(core->print, "Analyze all flags starting with sym.go. (aF @@f:sym.go.*)");
-					rz_core_cmd0(core, "aF @@f:sym.go.*");
-					rz_print_rowlog_done(core->print, oldstr);
-				}
-				rz_core_task_yield(&core->tasks);
-				if (!cfg_debug) {
-					if (dh_orig && strcmp(dh_orig, "esil")) {
-						rz_core_cmd0(core, "dL esil");
-						rz_core_task_yield(&core->tasks);
-					}
-				}
-				int c = rz_config_get_i(core->config, "analysis.calls");
-				rz_config_set_i(core->config, "analysis.calls", 1);
-				ut64 t = rz_num_math(core->num, "$S");
-				rz_core_seek(core, t, true);
-				if (rz_cons_is_breaked()) {
-					goto jacuzzi;
-				}
-
-				oldstr = rz_print_rowlog(core->print, "Analyze function calls (aac)");
-				(void)rz_cmd_analysis_calls(core, "", false, false); // "aac"
-				rz_core_seek(core, curseek, true);
-				// oldstr = rz_print_rowlog (core->print, "Analyze data refs as code (LEA)");
-				// (void) cmd_analysis_aad (core, NULL); // "aad"
-				rz_print_rowlog_done(core->print, oldstr);
-				rz_core_task_yield(&core->tasks);
-				if (rz_cons_is_breaked()) {
-					goto jacuzzi;
-				}
-
-				if (is_unknown_file(core)) {
-					oldstr = rz_print_rowlog(core->print, "find and analyze function preludes (aap)");
-					(void)rz_core_search_preludes(core, false); // "aap"
-					didAap = true;
-					rz_print_rowlog_done(core->print, oldstr);
-					rz_core_task_yield(&core->tasks);
-					if (rz_cons_is_breaked()) {
-						goto jacuzzi;
-					}
-				}
-
-				oldstr = rz_print_rowlog(core->print, "Analyze len bytes of instructions for references (aar)");
-				(void)rz_core_analysis_refs(core, ""); // "aar"
-				rz_print_rowlog_done(core->print, oldstr);
-				rz_core_task_yield(&core->tasks);
-				if (rz_cons_is_breaked()) {
-					goto jacuzzi;
-				}
-				if (is_apple_target(core)) {
-					oldstr = rz_print_rowlog(core->print, "Check for objc references");
-					rz_print_rowlog_done(core->print, oldstr);
-					cmd_analysis_objc(core, input + 1, true);
-				}
-				rz_core_task_yield(&core->tasks);
-				oldstr = rz_print_rowlog(core->print, "Check for vtables");
-				rz_core_cmd0(core, "avrr");
-				rz_print_rowlog_done(core->print, oldstr);
-				rz_core_task_yield(&core->tasks);
-				rz_config_set_i(core->config, "analysis.calls", c);
-				rz_core_task_yield(&core->tasks);
-				if (rz_cons_is_breaked()) {
+				bool experimental = true ? (input[1] == 'a') : false;
```

Comment:
?????

Reply:
I translated the logic of `aaaa` detection further in the function.

Reply:
I think you meant
```suggestion
				bool experimental = input[1] == 'a';
```